src/mame/mame.lst

```diff
@@ -41224,18 +41224,21 @@ twincobru                       // B30 / TP-011 (c) 1987 Taito America Corporati
 bmiidx                          //
 bmiidx2                         //
 bmiidx3                         //
+bmiidx3b                        //
 bmiidx3a                        //
```

Comment:
Please keep this file sorted, it makes it less unwieldy.

Reply:
Done.